src/main/java/org/sunbird/workflow/service/UserBulkUploadService.java

```diff
@@ -260,27 +244,90 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
                     if (nextRow.getCell(7) != null && nextRow.getCell(7).getCellType() != CellType.BLANK) {
-                        String payType  = null;
-                        if (nextRow.getCell(7).getCellType() == CellType.NUMERIC) {
-                            payType = NumberToTextConverter.toText(nextRow.getCell(7).getNumericCellValue());
-                        } else if (nextRow.getCell(7).getCellType() == CellType.STRING) {
-                            payType = nextRow.getCell(7).getStringCellValue().trim();
+                        if (nextRow.getCell(7).getCellType() == CellType.STRING) {
+                            valuesToBeUpdate.put(Constants.DOB, nextRow.getCell(7).getStringCellValue().trim());
                         } else {
-                            errList.add("Invalid Value of Grade Pay type. Expecting number/string format");
+                            errList.add("Invalid column type. Expecting string format");
                         }
-                        valuesToBeUpdate.put(Constants.PAY_TYPE, payType);
                     }
                     if (nextRow.getCell(8) != null && nextRow.getCell(8).getCellType() != CellType.BLANK) {
-                        valuesToBeUpdate.put(Constants.INDUSTRY, nextRow.getCell(8).getStringCellValue().trim());
+                        if (nextRow.getCell(8).getCellType() == CellType.STRING) {
+                            valuesToBeUpdate.put(Constants.DOMICILE_MEDIUM, nextRow.getCell(8).getStringCellValue().trim());
+                        } else {
+                            errList.add("Invalid column type. Expecting string format");
+                        }
                     }
                     if (nextRow.getCell(9) != null && nextRow.getCell(9).getCellType() != CellType.BLANK) {
-                        valuesToBeUpdate.put(Constants.LOCATION, nextRow.getCell(9).getStringCellValue().trim());
+                        if (nextRow.getCell(9).getCellType() == CellType.STRING) {
+                            valuesToBeUpdate.put(Constants.CATEGORY, nextRow.getCell(9).getStringCellValue().trim());
+                        } else {
+                            errList.add("Invalid column type. Expecting string format");
+                        }
+                    }
+                    if (nextRow.getCell(10) != null && nextRow.getCell(10).getCellType() != CellType.BLANK) {
+                        if (nextRow.getCell(10).getCellType() == CellType.NUMERIC) {
+                            valuesToBeUpdate.put(Constants.PINCODE, NumberToTextConverter.toText(nextRow.getCell(10).getNumericCellValue()));
+                        } else if (nextRow.getCell(10).getCellType() == CellType.STRING) {
+                            valuesToBeUpdate.put(Constants.PINCODE, nextRow.getCell(10).getStringCellValue().trim());
+                        } else {
+                            errList.add("Invalid column type. Expecting number/string format");
```

Comment:
Instead of "column" -- do provide which field we are trying to parse and the expected format.